Issue #75

AlexPernot opened the pull request:
Expanded the size of the arrow divs, and reajusted text alignment.
This way, the active zones of the arrows are bigger.

-- User @romainchalut also helped me.

Yoric commented on css/style.css:
```diff
@@ -23,6 +23,8 @@ html, body {
     position: absolute;
     bottom: 0;
     z-index: 5;
+	width: 20%;
```
Please fix the indentation.

Yoric commented:
This looks almost good. Can you fix the indentation before I merge your pull request?

AlexPernot commented:
Done!

Yoric commented:
I'd prefer a single commit. Can you fold them using `git rebase -i`?

Commit:
Issue #75

The active zone's size of the navigation arrows is bigger.

-- User @romainchalut also helped me.

AlexPernot commented:
Should be good now.

Yoric commented:
Looks good, thank you very much.
